kolibri/core/assets/test/views/NotificationsRoot.spec.js

```diff
@@ -1,13 +1,20 @@
 import { shallowMount } from '@vue/test-utils';
 import { UserKinds } from 'kolibri.coreVue.vuex.constants';
+import useUser, { useUserMock } from 'kolibri.coreVue.composables.useUser';
```

Comment:
I'm not totally clear on why this now has to be imported here. Is it because previously the user related stuff was in `...options` ?

I know we now have the useUser composable, and I understand we're changing the core module import in the spec. I just don't fully understand why the composable change is happening in the spec now. Did we just not update the tests before to use the composable?